spec/acceptance/mod_mime_spec.rb

```diff
@@ -32,6 +32,9 @@ class { 'apache': }
 
     describe file("#{mod_dir}/mime.conf") do
       it { is_expected.to contain "AddType application/x-compress .Z" }
+      it { is_expected.to contain "AddHandler type-map var" }
+      it { is_expected.to contain "AddType type-map .shtml" }
+      it { is_expected.to contain "AddOutputFilter INCLUDES .shtml" }
```

Comment:
to test for [https://github.com/puppetlabs/puppetlabs-apache/pull/1232/files#r43663096], i recommend you add a `\n` before that closing `"`…